src/components/callout/callout.stories.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+import Callout from './callout';
+import ShootingStar from '../../illustrations/shooting-star.svg';
+import ConstellationBigDipper from '../../illustrations/constellation-big-dipper.svg';
+
+const EXAMPLE_ILLUSTRATIONS = {
+  None: null,
+  ShootingStar: <ShootingStar />,
+  ConstellationBigDipper: <ConstellationBigDipper />,
+};
+
+export default {
+  component: Callout,
+  title: 'Components/Callout',
+  parameters: {
+    componentSubtitle:
+      'Display additional information detached from your surrounding text content',
+  },
+  argTypes: {
+    illustration: {
+      options: Object.keys(EXAMPLE_ILLUSTRATIONS),
+      mapping: EXAMPLE_ILLUSTRATIONS,
+      description:
+        'This is an example list of icons, there is not default set. You have to provide a svg react component/',
+      control: {
+        type: 'radio', // Type 'select' is automatically inferred when 'options' is defined
+        labels: {
+          // 'labels' maps option values to string labels
```

Comment:
For a splendidly clean code base I suggest we are very consistent with _everything_. In this case I'd suggest not to use mixed types of comments (at the end of the line for `type` and as a block for `labels`). Also the placement of the `labels`  comment seems slightly off. Why is it inside the object? I guess one could argue both ways... when in doubt I suggest to not use any comments at all. Most of this information is readily available in the documentation.

```suggestion
        // Type 'select' is automatically inferred when 'options' is defined
        type: 'radio',
        // 'labels' maps option values to string labels
        labels: {
```

Reply:
yes you are right. that is a leftover from where ever I copied it. I'm slowly coming back into "be careful" mode, especially with our now long-term components in storybook. 🙏 I will fix this, both comments are now obsolete.